test/middleware.verify-token.test.js

```diff
@@ -44,9 +44,9 @@ test.cb('VerifyToken() token option matching verify_token', t => {
   })
 })
 
-test('VerifyToken() token option matching verify_token', t => {
+test('VerifyToken() token option nonmatching verify_token', t => {
```

Comment:
It's probably confusing just looking at the diff. But there are two tests with identical names. This test is checking for a nonmatching token, so I just cleaned up the naming https://github.com/BeepBoopHQ/slapp/pull/51/files/ad7dbaa6b90800c2843d0aa0ba0e118b8b8e4c1b#diff-9bbe255a1eb0136834e562fbfff9e234L29